site/content/blog/2021-07-01-whats-new-in-svelte-july-2021.md

```diff
@@ -0,0 +1,56 @@
+---
+title: What's new in Svelte: July 2021
+description: Keeping cool with fixes, TypeScript tooling and tonnes of new features
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+As the northern hemisphere heats up, Svelte has stayed cool with lots of performance and bug fixes, better TypeScript support, and lots of new components & tools from around the ecosystem. Let's take a peek 👀
+
+## New in SvelteKit
+- `adapter-node` now precompresses assets using gzip & brotli ([#1693](https://github.com/sveltejs/kit/pull/1693))
+- Support for TypeScript transpilation has been added to the svelte package tooling ([#1633](https://github.com/sveltejs/kit/pull/1633))
+
+
+## Features & bug fixes from around svelte/*
+- [Svelte 3.38.3](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#3383) (released June 22) includes a bunch performance and bug fixes - including hydration optimizations, `this` preservation in bubbled events, and more!
+- The latest language tools releases added support for prop renaming from outside of a component, postcss syntax grammar, `.d.ts` file outputs in `svelte2tsx` and better support for TypeScript features across the board!
+- `svelte-scroller` got some quality-of-life fixes in 2.0.7 - fixing an initial width bug and updating its `index` more conservatively
+
+---
+
+## Community Showcase
+
+**Apps & Sites**
+- [SvelteThemes](https://sveltethemes.dev/) is a curated list of Svelte themes and templates built using svelte, sveltekit, elderjs, routify etc.
+- [Beatbump](https://github.com/snuffyDev/Beatbump) is an alternative frontend for YouTube Music created using Svelte/SvelteKit.
+- [Sveltuir](https://github.com/webspaceadam/sveltuir) is an app help you memorize the guitar fretboard
+
+
+**Educational Content**
```

Comment:
I wrote a [guest post for CSS-Tricks](https://css-tricks.com/using-custom-elements-in-svelte/) on Svelte and custom elements that would be nice to feature.

Possibly: 
- [Using Custom Elements in Svelte](https://css-tricks.com/using-custom-elements-in-svelte/) shows some of the quirks to look out for when using custom elements in a Svelte site.